lib/Stan_math_code_gen.ml

```diff
@@ -20,15 +36,17 @@ let%expect_test "emit function type raises" =
   | exception e -> print_s [%sexp (e : exn)] ) ;
```

Comment:
Maybe use a more informative name than `ad` in the above for the string that will represent the base type?